packages/components/src/mobile/readable-content-view/style.scss

```diff
@@ -0,0 +1,8 @@
+.container {
```

Comment:
Filename should be updated to `style.native.scss` to ensure it never gets included for the web version.

Reply:
Thanks, updated!